test/multi_config/resources/WindowsLogOnlyConfig.json

```diff
@@ -0,0 +1,132 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "logfile": "c:\\ProgramData\\Amazon\\AmazonCloudWatchAgent\\Logs\\amazon-cloudwatch-agent.log"
+  },
+  "metrics": {
+    "metrics_collected": {
+      "Processor": {
+        "measurement": [
+          "% Idle Time",
+           "% InterruptTime",
+          "% User Time",
+          "% Processor Time"
+        ],
+        "resources": [
+          "*"
+        ],
+        "metrics_collection_interval": 1
+      },
+      "LogicalDisk": {
+        "measurement": [
+          "% Free Space"
+        ],
+        "resources": [
+          "*"
+        ]
+      },
+      "PhysicalDisk": {
```

Comment:
This JSON file has a misleading name. It is called `WindowsLogOnlyConfig.json`, but I see a lot of metrics in it.